src/main/java/seedu/address/logic/commands/MarkCommand.java

```diff
@@ -26,16 +27,19 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Marks the task identified by the index number used in the displayed task list as completed.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Parameters: INDEX[ES] (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1, " + COMMAND_WORD + " 1 2 3";
 
-    public static final String MESSAGE_MARK_TASK_SUCCESS = "Completed Task: %1$s";
     public static final String MESSAGE_TASK_ALREADY_COMPLETED = "This task is already marked as complete.";
 
-    private final Index targetIndex;
+    private final ArrayList<Index> targetIndexes;
```

Comment:
It might provide a higher level of abstraction if we use List instead of ArrayList (we can still assign an ArrayList to it, but this hides implementation detail).